### ISSUE TITLE
use branch master and feature/tlsv1.3 to scan the same ip+port, the results of secure_renegotiation and scts are different

### ISSUE THREAD

**Issue by chushuai** [zmap/zcrypto#342]
Use branch master and feature/tlsv1.3 to scan the same ip+port, The results of secure_renegotiation and scts are different 

```
func TestTls(t *testing.T){
	tests := []string{ "138.201.124.182:10250"}
	for _, test := range tests {
		conn, err := Dial("tcp", test, &Config{
			InsecureSkipVerify: true,
		})
		data, _ := json.Marshal(conn.GetHandshakeLog())
		fmt.Println(string(data))
		if err != nil {
			fmt.Println("failed to connect: " + err.Error())
		}
	}

}
```

![image](https://user-images.githubusercontent.com/5875458/181745057-e9fef321-735f-407a-bdd2-ad11a8a343ef.png)

**Comment by mzpqnxow** [comment]
The tls1.3 branch includes tls1.3 support and also includes other enhancements

This includes (at least) early renegotiation handling

You can see part of this if you track the zcrypto portion of https://github.com/zmap/zgrab2/pull/334

tl; dr; this is expected behavior as far as I know

**Comment by mzpqnxow** [comment]
Is this still an issue or can it be resolved?

I admit that only a year later, I've forgotten most of the details and I'm not sure I even understand my own response..

I do know that TLS1.3 has been satisfactory for me since zgrab2 started using it in master, so maybe there's nothing to do here?

🤞

**Comment by phillip-stephens** [comment]
I think post merge of the TLS 1.3 branch into `master`, this can be closed. This doesn't seem to be a regression as much as a change in behavior.

cc @developStorm
